NYCRunawayAndHomelessYouth/nyc_runaway_and_homeless_youth.py

```diff
@@ -0,0 +1,64 @@
+import os
+import sys
+from datetime import datetime
+import re
+import numpy as np
+import requests
+from bs4 import BeautifulSoup
+import pandas as pd
+from pymongo import MongoClient, errors
+from tqdm import tqdm
+
+_i = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+if _i not in sys.path:
+    # add parent directory to sys.path so utils module is accessible
+    sys.path.insert(0, _i)
+del _i  # clean up global name space
+from shared_code.utils import (
+    check_similarity, locate_potential_duplicate,
+    insert_services, client
+)
+from shared_code.base_scraper import BaseScraper
+
+
+class NYC_Youth_Scraper(BaseScraper):
+
+
+    def grab_data(self) -> pd.DataFrame:
+
+        df = super().grab_data()
+        df['state'], df['city'] = 'NY', 'New York'
```

Comment:
City can be `New York City`

Reply:
Can we force the zipcode to be strictly first 5 digit, in case it has more digits.